Plots/Contours/NCL_ce_1.py

```diff
@@ -0,0 +1,95 @@
+"""
+NCL_ce_1.py
+===========
+This script illustrates the following concepts:
+   - Drawing black-and-white contours over a cylindrical equi-distant map
+
```

Comment:
```suggestion
   - Specifying contour labels 
   
```
This is with respect to my comment below about NCL being annoying